bigtable/tests/system.py

```diff
@@ -84,7 +84,7 @@ def setUpModule():
         credentials = EmulatorCreds()
         Config.CLIENT = Client(admin=True, credentials=credentials)
     else:
-        Config.CLIENT = Client(admin=True)
+        Config.CLIENT = Client(project='grass-clump-479', admin=True)
```

Comment:
Please stop injecting your local project here:  this value should be in your credentials, or in the environment.

Reply:
@tseaver: do you have a reference to instructions on how to set up an environment for system tests via credentials or in the environment?

Reply:
@sduskis [Sure](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/CONTRIBUTING.rst#running-system-tests)